mem_sql/src/main/java/com/splicemachine/derby/lifecycle/MemDatabase.java

```diff
@@ -59,6 +59,7 @@ public static void main(String...args) throws Exception{
             System.setProperty("com.splicemachine.enableLegacyAsserts",Boolean.TRUE.toString());
             SanityManager.DEBUG_SET("DumpOptimizedTree");
         }
+        System.setProperty("com.splicemachine.isMemPlatform",Boolean.TRUE.toString());
```

Comment:
Remove this since we don't rely on it anymore.

Reply:
Thanks @dgomezferro , I've removed the line.